doc/developer-notes.md

```diff
@@ -259,14 +259,15 @@ on all categories (and give you a very large `debug.log` file).
 The Qt code routes `qDebug()` output to `debug.log` under category "qt": run with `-debug=qt`
 to see it.
 
-### Testnet and Regtest modes
+### Signet, testnet, and regtest modes
```

Comment:
```suggestion
### Test chains
```

nit?

Reply:
thanks! no strong opinion about chains :chains: :chains: but reckoned @kallewoof and @ajtowns would relish seeing "Signet" in the section header :mega: